doc/developer-guide.md

```diff
@@ -253,10 +250,9 @@ The builtin CMake options used when invoking CMake on the command line.
   relevant llvm headers and support tools, and their version must match
   a supported LLVM version.
 
-* `CA_ENABLE_API`: Semi-colon separated list of APIs to enable. Valid values
-  are `cl` for OpenCL, and `vk` for Vulkan. Enabling an API when an optional
-  component is not present dependent on license agreement will result in a CMake
-  error. The default is `cl;vk`.
+* `CA_ENABLE_API`: Semi-colon separated list of APIs to enable. Valid values are
+  `cl` or '' for OpenCL. This currently only supports OpenCL, but that may change in
+  the future.
```

Comment:
If we don't have any plans to add an API other than `cl`, I'm not sure we should suggest that we're considering that.